pkg/cluster/cluster_info_scraper.go

```diff
@@ -318,30 +324,33 @@ func (s *ClusterScraper) GetPodGrandparentInfo(pod *api.Pod) (string, string, st
 			Resource: commonutil.ReplicationControllerResName}
 	case commonutil.KindReplicaSet:
 		res = schema.GroupVersionResource{
-			Group:    commonutil.K8sAPIReplicasetGV.Group,
-			Version:  commonutil.K8sAPIReplicasetGV.Version,
+			Group:    commonutil.K8sAPIDeploymentGV.Group,
+			Version:  commonutil.K8sAPIDeploymentGV.Version,
```

Comment:
Looks we need `K8sAPIReplicasetGV` here instead of `K8sAPIDeploymentGV`. Or is there a reason to use Deployment GroupVersion to get ReplicaSet resource?

Reply:
Right. I guess this was introduced while rebasing my changes on latest. Updated!